library/src/test/java/bio/terra/pfb/PfbReaderTest.java

```diff
@@ -0,0 +1,86 @@
+package bio.terra.pfb;
+
+import static bio.terra.pfb.utils.CompareOutputUtils.FileExtension.JSON;
+import static bio.terra.pfb.utils.CompareOutputUtils.FileExtension.TXT;
+import static bio.terra.pfb.utils.CompareOutputUtils.PfbCommandType.*;
+
+import bio.terra.pfb.utils.CompareOutputUtils;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+class PfbReaderTest {
+
+  private static final List<String> listOfTestFiles =
+      List.of("minimal_schema", "test", "kf", "data-with-array", "empty");
+
+  @Test
+  void showSchemaTest() throws IOException {
+    // TODO - remove when fix is added for AJ-1288
+    var editedListOfTestFiles = listOfTestFiles.stream().filter(f -> !f.equals("empty")).toList();
```

Comment:
What's the significance of the `empty` file here, and why does it need to be excluded?

Reply:
Definitely confusing -- "Empty" is the name of a test file that is definitely not empty. I'll rename it. 
But, this showSchema test fails for the test file "empty" and the root issue will be address in AJ-1288.